### ISSUE TITLE
J-20 ＆ FC-31 Radar signatures correction

### ISSUE THREAD

**Issue by LuigiP57** [PygmalionOfCyprus/cmo-db-requests#1712]
### DB Selector

DB3K

### Affected DBID(s)

aircraft #2463 #5012 #5013 #5014 #5454 #2917 #4454

### Summary of Changes

There are some mistakes in those two stealth fighters' radar signatures, IMO a little bit corrections might needed.

[For J-20]
Radar A-D Band: change to "front: -20.00 dBsm, 0.010 sqm; side: -16.00 dBsm, 0.020 sqm; rear: -20.00 dBsm, 0.010 sqm"
Radar A-D Band: change to "front: -30.00 dBsm, 0.0010 sqm; side: -26.00 dBsm, 0.0020 sqm; rear: -30.00 dBsm, 0.0010 sqm"

[For FC-31]
Same as J-20

### Sources

http://www.ausairpower.net/APA-2011-03.html
https://www.alternatewars.com/BBOW/Radar/Decibels_Radars.htm

**Comment by LuigiP57** [comment]
![image](https://user-images.githubusercontent.com/107970049/178290277-3b32b735-d838-45ed-a58e-91fab105222f.png)
J-20 Specular RCS Model Results With RAM / CFC in 3.0 GHz
![image](https://user-images.githubusercontent.com/107970049/178290471-28d7f95a-0bbd-45d7-8be5-97cff213a129.png)
J-20 Specular RCS Model Results With RAM / CFC in 16.0 GHz

Quote from the article: "This study has therefore established through Physical Optics simulation across nine radio-frequency bands, that no fundamental obstacles exist in the shaping design of the J-20 prototype precluding its development into a genuine Very Low Observable design."

**Comment by LuigiP57** [comment]
More intuitive: 
![image](https://user-images.githubusercontent.com/107970049/178291056-893036c1-f4f3-46d3-9a25-6994b824f2b2.png)

![image](https://user-images.githubusercontent.com/107970049/178290997-c52c4881-66fe-40df-85e0-e5452110b0ef.png)

**Comment by PygmalionOfCyprus** [comment]
I hope you'll forgive me for not writing a full rebuttal of this in the interest of time. 

I actually reviewed this report, among with many others, when determining the new stealth signatures. In fact, the J-20 was one of my "baseline" aircraft (for obvious reasons, namely its importance to the modern security environment). I did a lot of research and put a fair bit of work into trying to pin down its "degree" of stealthiness. I won't pretend the RCS values in the DB are perfect, but I do feel confident in its "ranking" when compared to neer-peers. The values you propose would make the J-20 equivalent in stealth to the F-22; this is not the case.

First, a note on ethos: no discussion of the APA study can be had without recognizing their heavy skew towards Russian/Chinese equipment. (Kopp especially has a tendency to underestimate and undeservedly belittle the F-35.) Don't get me wrong: APA does good work. But one must balance for bias.

This line, for example, is laughable; Kopp should know better, even a decade ago:

>Overall, the stealth shaping of the J-20 prototype design is without doubt considerably better than that seen in the Russian T-50 PAK-FA prototypes and, even more so, than that seen in the intended production configuration of the United States' F-35 Joint Strike Fighter.

We also have to remember that the APA study, released over a decade ago, is now quite dated. With the benefit of hindsight we can identify numerous ways in which its conclusions do not reflect reality.

Finally, it also suffers from the same issues as any commercial/private RCS analysis. RCS modelling is a highly flawed "art". It rarely reflects reality. The APA have done a good job listing their assumptions in creating the model: at every stage they have assumed a "best possible case." Even a perfect model -- which privately-generated models created from public imagery are not -- takes into account only shaping. There is a lot more to stealth than shaping, and without knowing the efficiency of Chinese RAM, etc. any findings of the simulation are based solely on a single piece of the puzzle.

To give credit where credit is due, the study does itself make note of this:

>The assessment cannot be more than preliminary for a number of good reasons:
>1. The final airframe shaping remains unknown, and changes may arise through the development cycle, to improve aerodynamic performance, operational characteristics, and LO/VLO performance;
>2. The state of Chinese Radar Absorbent Materials (RAM), Radar Absorbent Structures (RAS) and radar absorbent coatings technology is not well understood in the West;
>3. The state of Chinese technologies for sensor aperture (radar, EO, passive RF) structural mode RCS reduction is not well understood in the West;
>4. The state of Chinese technologies for RCS flare spot reduction, in areas such as  navigation/communications antennas, seals, panel joins, drain apertures, cooling vents, and fasteners is not well understood in the West.

(This is where the "interest of time" comes in; you're going to have to accept a truncated explanation here.)

It is well accepted that Chinese RAM, RCS reduction, etc. are years -- potentially decades -- behind the West. That the J-20 is indeed competently shaped (at least ignoring the canards, which despite the APA's attempt to handwave do have a significant degrading effect on stealth) does not make it equivalent to the F-22 or F-35, which have decades of experience in RA technology behind them.

Does this mean that the Chinese will never create a competitive stealth fighter? No, of course not. Their work on the J-20 is very impressive and deserves praise. But as their first production stealth aircraft, the J-20 -- while certainly stealthy-_looking_ and thereby very conducive to aggressive propaganda claiming incredible stealth features -- does not have everything it needs "under the hood" to be in the same tier as the F-22/F-35.

For this reason it exists in its current modifier tier.

![image](https://user-images.githubusercontent.com/87087226/178498150-eb7f4d9a-bfda-4174-9cd1-b672fe0f07e4.png)

In practice, this means the J-20 is a fearsome foe to anyone not in an F-22 or an F-35. Even then, given some luck and clever maneuvering, a J-20 has a chance to snag a kill -- not a great chance, but a chance, which is more than can be said for most other aircraft. I think this is most reflective of its place in the stealth hierarchy.
